code/modules/unit_tests/limbsanity.dm

```diff
@@ -10,3 +10,31 @@
 				Fail("[path] does not have a valid icon for female variants")
 		else if(!icon_exists(UNLINT(part.should_draw_greyscale ? part.icon_greyscale : part.icon_static), "[part.limb_id]_[part.body_zone]"))
 			Fail("[path] does not have a valid icon")
+
+/datum/unit_test/limb_hight_adjustment
+
+/datum/unit_test/limb_hight_adjustment/Run()
+	var/mob/living/carbon/human/john_doe = allocate(/mob/living/carbon/human/consistent)
+	var/mob/living/carbon/human/species/monkey/monkey = allocate(/mob/living/carbon/human/species/monkey)
```

Comment:
Don't use the `species` things, especially for "tallboy" because they are not consistent. Use `consistent` and call set_species manually